src/Products/PluggableAuthService/tests/test_exportimport.py

```diff
@@ -275,7 +275,7 @@ def test_empty_adding_plugins(self):
         csv_plugin = pas._getOb('csv_plugin')
         self.assertTrue(csv_plugin.__class__ is TestCSVAware)
         self.assertEqual(csv_plugin._was_put.getvalue().strip(),
-                         KNOWN_CSV.strip())
+                         KNOWN_CSV.encode('ascii').strip())
```

Comment:
This is OK as `csv_plugin` contains bytes (`_was_put` is a `BytesIO`). To be more consistent with all other places that manually encode strings I'd use UTF-8, though.

```suggestion
                         KNOWN_CSV.encode('UTF-8').strip())
```